controllers/datafeed_controller.py

```diff
@@ -107,9 +107,20 @@ class FMSAPIEventAlliancesEnqueue(webapp.RequestHandler):
     Handles enqueing getting alliances from the FMS API
     """
     def get(self, when):
+        events = skipped_events = []
         if when == "now":
             events = EventHelper.getEventsWithinADay()
             events = filter(lambda e: e.official, events)
+
+            # Check if we should do a full query, or temporarily skip events
+            # that are not ending today. (All events will be checked on
+            # half-hour intervals, and events ending today whenever the cron
+            # job runs.)
+            minutes_from_full = (((int(time.time()) / 60) + 15) % 30) - 15
```

Comment:
I agree with Greg for having two cron jobs.

Maybe the two endpoints could be something along the lines of:
`/tasks/enqueue/fmsapi_event_alliances/now`
`/tasks/enqueue/fmsapi_event_alliances/last_day_only`

Every 30 minutes we would end up running the cron jobs twice, but that's not too big of an issue IMO.